### ISSUE TITLE
more specific compiler warning string grep

### ISSUE THREAD

**Issue by nickdesaulniers** [kernelci/kernelci-core#34]
for kernelCI, I see reports like:

```
Warnings summary:

    111  1 warning generated.
    14   2 warnings generated.
    7    drivers/net/ethernet/intel/iavf/iavf_osdep.h:49:18: warning: 'format' attribute argument not supported: gnu_printf [-Wignored-attributes]
...
    4    3 warnings generated.
    3    4 warnings generated.
...
   2    drivers/char/hw_random/optee-rng.c:177:31: warning: suggest braces around initialization of subobject [-Wmissing-braces]
    2    5 warnings generated.
    2    41 warnings generated.
```

when grepping logs for compiler warnings, I find that grepping for `warning:`, specifically with the end colon, helps filter to just the warnings.  This would remove the count of warnings per file from the logs, which I think would be a little clearer.  Also, I'm not sure if the "warnings generated" lines are polluting the total warning count at the top of the report? cc @khilman @broonie

**Comment by nickdesaulniers** [comment]
obsoleted by: https://github.com/kernelci/kernelci-backend/pull/144?

**Comment by gctucker** [comment]
This issue is about build reports, whereas https://github.com/kernelci/kernelci-backend/pull/144 is about test reports.  So it still needs to be resolved.

**Comment by mgalka** [comment]
@gctucker @JenySadadia Is it something that should be addressed with the new pipeline?

**Comment by gctucker** [comment]
Yes, right now the kernel builds don't parse the logs but we should be doing something about it at some point to report warnings etc.

**Comment by gctucker** [comment]
I think however that this issue should be closed, and instead we should have a new issue as part of the backlog to implement kernel build log parsing.

**Comment by padovan** [comment]
legacy material. New kernelci has logspec for compiler error matching.
